src/01-functions.js

```diff
@@ -112,7 +124,13 @@ function averageThree() {}
  *  compareTypes("left", 10);
  *  //> false
  */
-function compareTypes() {}
+function compareTypes(a, b) {
+  if(typeof a === typeof b){
+    return true;
+  }else{
```

Comment:
John!  Great looking code - in fact there were no issues before line 130.  Lets space out our else  => ` } else { ` like so